packages/backend/src/peripherals/database/BaseRepository.ts

```diff
@@ -26,6 +26,10 @@ interface DeleteMethod<A extends unknown[]> {
   (...args: A): Promise<number>
 }
 
+interface SaveMethod<T> {
```

Comment:
Maybe we can call it `UpdateMethod`? What do you think?

Reply:
update would take a partial - save updates the whole record.